R/frequencies.R

```diff
@@ -548,8 +548,8 @@ freqs_plot <- function(df, ..., top = 10, rm.na = FALSE, abc = FALSE,
   }
 
   p <- (p1 / p2)
-  attr(p, "data") <- aux
-  attr(p, "labels") <- labels
+  # attr(p, "data") <- aux
+  # attr(p, "labels") <- labels
```

Comment:
I don't know how these attributes were being used before. ggplot2 now has S7 properties with the same name and S7 properties are implemented as attributes, so these disrupt the plot structure. If they do anything important, you may have to find a workaround.